command/agent_test.go

```diff
@@ -61,191 +61,6 @@ func testAgentCommand(tb testing.TB, logger hclog.Logger) (*cli.MockUi, *AgentCo
 	}
 }
 
-/*
-func TestAgent_Cache_UnixListener(t *testing.T) {
```

Comment:
I'm not sure what this is, so I deleted it as it was commented out. We can always fish it out of git if we need it again, but I don't like having a ~200 line comment block in files like this.